pubspec.yaml

```diff
@@ -10,8 +10,8 @@ dependencies:
   flutter_local_notifications: any
   image_gallery_saver: any
   image_picker: any
+  meta: ^1.2.0
   path_provider: any
-  pedantic: any
```

Comment:
@jakemac53 this line in particular was problematic - for whatever reason, with it left as is, the pubspec.lock controlled a too-low version of pedantic.

Reply:
Did you try a `pub upgrade` (or one of the variations of `flutter packages pub upgrade`)?

The pubspec.lock locks your versions to exactly those versions, so checking it in pins you to exactly those until you run a `pub upgrade`.

Reply:
`flutter pub upgrade` does fix the unawaited issue, but causes mayhem with other packages because they're all using the any constraint.  So instead of one analysis error, there are 21 after a pub upgrade in this project :)